pkgs/development/tools/misc/sccache/default.nix

```diff
@@ -13,7 +13,7 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "1cfdwf00jgwsv0f72427asid1xr57s56jk5xj489dgppvgy7wdbj";
 
-  cargoBuildFlags = [ "--features=all" ];
+  cargoBuildFlagsArray = [ "--features=dist-client dist-server" ];
```

Comment:
Comma separated is how this is handled currently.

```suggestion
  cargoBuildFlags = [ "--features=dist-client,dist-server" ];
```

Reply:
aaaaaah. So no need for spaces ever?

Reply:
Probably not, the [cargo build docs](https://doc.rust-lang.org/cargo/commands/cargo-build.html#feature-selection) says both are valid.

Reply:
reworked. thank you!